infra/build.gradle.kts

```diff
@@ -71,28 +75,28 @@ dependencies {
         exclude("it.geosolutions.jgridshift", "jgridshift-core")
     }
     implementation("org.apache.commons:commons-csv:1.10.0")
-    implementation("commons-io:commons-io:2.15.1")
+    implementation("commons-io:commons-io:2.16.1")
     implementation("com.auth0:jwks-rsa:0.22.1")
     implementation("com.auth0:java-jwt:4.4.0")
-    implementation("io.netty:netty-resolver-dns-native-macos:4.1.107.Final:osx-aarch_64")
-    implementation("org.postgresql:postgresql:42.7.1")
-    implementation("jakarta.activation:jakarta.activation-api:2.1.2")
-    implementation("jakarta.xml.bind:jakarta.xml.bind-api:4.0.1")
+    implementation("io.netty:netty-resolver-dns-native-macos:4.1.109.Final:osx-aarch_64")
+    implementation("org.postgresql:postgresql:42.7.3")
+    implementation("jakarta.activation:jakarta.activation-api:2.1.3")
+    implementation("jakarta.xml.bind:jakarta.xml.bind-api:4.0.2")
     implementation("com.github.davidmoten:rtree2:0.9.3")
     implementation("commons-validator:commons-validator:1.8.0") {
         exclude("commons-logging", "commons-logging")
+        exclude("commons-collections", "commons-collections")
     }
     compileOnly("org.springframework.boot:spring-boot-devtools")
-    runtimeOnly("org.glassfish.jaxb:jaxb-runtime:4.0.4")
+    runtimeOnly("org.glassfish.jaxb:jaxb-runtime:4.0.5")
     annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion")
-    testImplementation("org.seleniumhq.selenium:selenium-java:4.18.0")
+    testImplementation("org.seleniumhq.selenium:selenium-java:4.19.1")
     //Do not update to version 5.15.0 as it causes StackOverflowError.
     //See: https://github.com/mock-server/mockserver/issues/1660
-    testImplementation("org.mock-server:mockserver-netty-no-dependencies:5.14.0")
```

Comment:
Tämä oli ongelman aiheuttaja. Oletan että syy menee jotenkin näin: tuo no-dependencies versio sisältänee .jar:ssa mukana kaiken tarvittavan, mukaanlukien logitoteutukset. Ne päätyivät voittamaan classpathissa springin kautta tulevan logitoteutuksen ja tuo aiheutti ongelman. Suora slf4j2-impl lisäys testipuolelle toi mukaan vielä kolmannen kirjastoversion ja ilmeisesti se puolestaan päätyi voittamaan tämän ja kaikki taas toimikin. Korjauksena, vaihdetaan mockserver tavalliseen versioon jolloin sen depsut ovat erillisiä kirjastoja ja dependency managementin pitäisi hoitaa classpathiin vain yksi versio. Ainakin minulla tuntuisi toimivan, mutta älähtäkää jos tuota vielä näkyy.